DTDL/publication_public.json

```diff
@@ -0,0 +1,28 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:io:tributech:publication:public;1",
```

Comment:
The properties `Disabled` and `Requests` are common with `dtmi:io:tributech:publication:restricted;1` and should be moved to the base.

Reply:
request info should only be available on the owing node same for soft delete

Reply:
@mayrmax @chlumper 
Should we add `OwingNode` as a property on `dtmi:io:tributech:publication:base;1` to resolve that issue? I think in general it would be nice to be able to interpret twins without the necessity of the context information of the "current" node.